lib/java/com/google/android/material/textfield/res/color/mtrl_filled_background_color.xml

```diff
@@ -16,6 +16,7 @@
 -->
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   <item android:alpha="0.16" android:color="?attr/colorOnSurface" android:state_hovered="true"/>
+  <item android:alpha="0.12" android:color="?attr/colorOnSurface" android:state_focused="true"/>
```

Comment:
I left the opacity the same as the default case just to keep the visuals consistent with current source.  Note that this bug (https://issuetracker.google.com/issues/68029303) means that I had to put this above the `state_enabled="false"` item.

Reply:
Is this change already released?

Reply:
If you check the full changelists under Releases, it looks like `1.1.0-beta02` didn't get it, but `1.2.0-alpha01` has it.

Reply:
Thanks. It would be great if we can set another state colors, like custom states or error state.

Reply:
I mean, for filledBox background color.